src/test/resources/testbib/Chocolate.bib

```diff
@@ -0,0 +1,225 @@
+  doi          = {10.3945/ajcn.111.023457},
+  issn         = {0002-9165},
+  number       = {3},
+  pages        = {740--751},
+  volume       = {95},
+  publisher    = {Elsevier BV},
+  readstatus   = {skimmed},
+}
+
+@Article{Di_Renzo_2012,
+  author       = {Di Renzo, Gian Carlo and Brillo, Eleonora and Romanelli, Maila and Porcaro, Giuseppina and Capanna, Federica and Kanninen, Tomi T and Gerli, Sandro and Clerici, Graziano},
+  date         = {2012-06},
+  journaltitle = {The Journal of Maternal-Fetal &amp; Neonatal Medicine},
+  title        = {Potential effects of chocolate on human pregnancy: a randomized controlled trial},
+  doi          = {10.3109/14767058.2012.683085},
+  issn         = {1476-4954},
+  number       = {10},
+  pages        = {1860--1867},
+  volume       = {25},
+  groups       = {Used},
+  publisher    = {Informa UK Limited},
+  readstatus   = {read},
+}
+
+@Article{Richard_2017,
+  author       = {Richard, Anna and Meule, Adrian and Friese, Malte and Blechert, Jens},
+  date         = {2017-09},
+  journaltitle = {Frontiers in Psychology},
+  title        = {Effects of Chocolate Deprivation on Implicit and Explicit Evaluation of Chocolate in High and Low Trait Chocolate Cravers},
+  doi          = {10.3389/fpsyg.2017.01591},
+  issn         = {1664-1078},
+  volume       = {8},
+  groups       = {Used},
+  publisher    = {Frontiers Media SA},
+  readstatus   = {read},
+}
+
+@Article{Fulton_1969,
+  author       = {Fulton, James E., Jr. and Plewig, Gerd and Kligman, Albert M.},
+  date         = {1969-12},
+  journaltitle = {JAMA: The Journal of the American Medical Association},
+  title        = {{Effect of Chocolate on Acne Vulgaris}},
+  doi          = {10.1001/jama.1969.03160370055011},
+  eprint       = {https://jamanetwork.com/journals/jama/articlepdf/350738/jama\_210\_11\_011.pdf},
+  issn         = {0098-7484},
+  number       = {11},
+  pages        = {2071-2074},
+  url          = {https://doi.org/10.1001/jama.1969.03160370055011},
+  volume       = {210},
+  abstract     = {{To test the widespread idea that chocolate is harmful in instances of acne vulgaris, 65 subjects with moderate acne ate either a bar containing ten times the amount of chocolate in a typical bar, or an identical-appearing bar which contained no chocolate. Counting of all the lesions on one side of the face before and after each ingestion period indicated no difference between the bars. Five normal subjects ingested two enriched chocolate bars daily for one month; this represented a daily addition of the diet of 1,200 calories, of which about half was vegetable fat. This excessive intake of chocolate and fat did not alter the composition or output of sebum. A review of studies purporting to show that diets high in carbohydrate or fat stimulate sebaceous secretion and adversely affect acne vulgaris indicates that these claims are unproved.}},

```

Comment:
is that paper open access? because abstracts tend to be copyrighted

Reply:
They are not all open access. I copied the bibtex data from scholar.google.de, then imported into JabRef, then with JabRef looked up the DOI, then imported data from DOI. There were two entries left without DOI, so I looked them up and imported the bibtex data from the journals website, which in this case included the abstract.

Reply:
fine, then just remove the abstract